Jenkinsfile

```diff
@@ -0,0 +1,21 @@
+/*This file uses jenkins shared library `vivarium_build_utils`,
+found at https://github.com/ihmeuw/vivarium_build_utils
+Due to Jenkins convention, importable modules must be stored
+in the 'vars' folder.
+Jenkins needs to be configured globally to use the correct branch.
+To configure the repo/branch go to:
+* Manage Jenkins
+  * Configure System
+    * Global Pipeline Libraries section
+      * Library subsection
+        * Name: The Name for the lib
+        * Version: The branch you want to use. Throws an error
+                   for nonexistent branches.
+        * Project Repository: Url to the shared lib
+        * Credentials: SSH key to access the repo
+
+Updating the shared repo will take affect on the next pipeline invocation.
+The "_" denotes that all modules will be imported from the shared library.
+*/ 
+@Library("vivarium_build_utils") _
+reusable_pipeline(scheduled_branches: ["release-candidate/v.orage.rebased"], upstream_repos: ["layered_config_tree"])
```

Comment:
typo "orage"

Reply:
Do you also need to pass in `python_versions: ["3.10", "3.11"]`? (we're doing that in vivarium, which I'm comparing this PR to)

Reply:
Vivarium does not have python versions in the jenkins file

Reply:
If you don't provide a `python_versions` argument it uses the default versions in VBU, which are 3.10 and 3.11